src/figuresClasses.ts

```diff
@@ -1,11 +1,66 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape = 'triangle';
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Sides of the triangle must be greater than 0');
+    }
+    if (a >= b + c || b >= a + c || c >= a + b) {
+      throw new Error(`Sides ${a}, ${b}, and ${c} can't form a triangle`);
+    }
```

Comment:
[CODE STYLE] - While throwing errors, it is beneficial to provide detailed descriptions in the error message. The current messages are good, but you might consider including the invalid values in the message for better debugging. For example: 'The side lengths ${a}, ${b}, and ${c} do not satisfy the triangle inequality theorem and cannot form a triangle.'